src/Domain/CoordinateMaps/TimeDependent/Translation.cpp

```diff
@@ -113,34 +193,64 @@ Translation<Dim>::jacobian(
     const std::unordered_map<
         std::string, std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>&
         functions_of_time) const {
-  if (f_of_r_ == nullptr) {
-    return identity<Dim>(dereference_wrapper(source_coords[0]));
-  } else {
-    std::array<tt::remove_cvref_wrap_t<T>, Dim> distance_to_center{};
-    for (size_t i = 0; i < Dim; i++) {
-      gsl::at(distance_to_center, i) =
-          gsl::at(source_coords, i) - gsl::at(center_, i);
-    }
-    auto radius = magnitude(distance_to_center);
+  // If an inner radius is not specified then take the jacobian of the
+  // MathFunction specific translation.
+  if (!inner_radius_.has_value()) {
+    if (f_of_r_ == nullptr) {
+      return identity<Dim>(dereference_wrapper(source_coords[0]));
+    } else {
+      std::array<tt::remove_cvref_wrap_t<T>, Dim> distance_to_center{};
+      for (size_t i = 0; i < Dim; i++) {
+        gsl::at(distance_to_center, i) =
+            gsl::at(source_coords, i) - gsl::at(center_, i);
+      }
+      auto radius = magnitude(distance_to_center);
+      const DataVector function_of_time =
+          functions_of_time.at(f_of_t_name_)->func(time)[0];
 
+      auto result = make_with_value<
+          tnsr::Ij<tt::remove_cvref_wrap_t<T>, Dim, Frame::NoFrame>>(
+          dereference_wrapper(source_coords[0]), 0.0);
+      for (size_t i = 0; i < Dim; i++) {
+        for (size_t j = 0; j < Dim; j++) {
+          for (size_t k = 0; k < get_size(radius); k++) {
+            if (get_element(radius, k) > 1.e-13) {
+              result.get(i, j) =
+                  (*f_of_r_).first_deriv(get_element(radius, k)) *
+                  gsl::at(function_of_time, i) *
+                  gsl::at(distance_to_center, j) / get_element(radius, k);
+            } else {
+              result.get(i, j) =
+                  (*f_of_r_).second_deriv(get_element(radius, k)) *
+                  gsl::at(function_of_time, i) * gsl::at(distance_to_center, j);
```

Comment:
The no-inner-radius case is for when you're using the mathfunction translation instead of the piecewise translation. I think this map will be smooth since the mathfunction provided will be a gaussian with a max at the center map argument . Also this limit is taken from `/spec/Dust/Domain/SpatialCoordMaps/SpatiallyDependentTranslation.cpp` lines 304-320

Reply:
Just because SpEC uses the wrong equation doesn't mean we should too.  I'll file an issue over there.

But erroring out if the map isn't smooth is probably the right thing to do, anyway.  I don't know where the radial function comes from, so I don't know  if it should be an ASSERT or an ERROR.

Reply:
Okay, so then I think the correct thing to do is`ASSERT(equal_within_roundoff(*f_of_r_).first_deriv(get_element(radius, k)), 0.)` then make the jacobian at that point 0.0